Rebuy-Forms-Rating.nuspec

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>Rebuy-Forms-Rating</id>
+    <title>RatingView for Xamarin.Forms</title>
+    <version>1.0.0</version>
+    <authors>Fokke Vermeulen</authors>
+    <owners>reBuy reCommerce GmbH</owners>
+    <licenseUrl>https://github.com/rebuy-de/Rebuy-Forms-Rating/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/rebuy-de/Rebuy-Forms-Rating</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Rebuy-Forms-Rating is a Xamarin.Forms view to let users rate your app in the appstores. Please check the website for further details.</description>
+    <summary>Rebuy-Forms-Rating is a Xamarin.Forms view to let users rate your app in the appstores.</summary>
+    <releaseNotes>1.0.0:
+* New: Opens an actionSheet to let user rate your app in the appstores.
+</releaseNotes>
+    <tags>rating rate RebuyFormsRating AppStore PlayStore apple google xamarin xamarin.forms rebuy unicorns</tags>
+    <dependencies>
+        <group>
+            <dependency id="Xamarin.Forms" version="1.5.0.6447" />
+        </group>
+
+        <group targetFramework="Acr.UserDialogs">
+            <dependency id="Acr.UserDialogs" version="4.2.0" />
+        </group>
+    </dependencies>
+  </metadata>
+  <files>
```

Comment:
Is _Splat_ a transitive or direct dependency? Just wondering, because if it's a direct dependency then I think it should be added here, if not, then ignore.